lib/bundle/dsl.rb

```diff
@@ -59,11 +59,22 @@ def mas(name, options = {})
       @entries << Entry.new(:mac_app_store, name, id: id)
     end
 
-    def tap(name, clone_target = nil)
```

Comment:
I wonder if it's possible to do `name, clone_target = nil, pin: false` or similar to avoid some of the logic below. @reitermarkus, any thoughts?

Reply:
I think `name, clone_target = nil, pin: false` should work fine.

Reply:
Yeh, that'd be good thanks @reitermarkus, @mikesplain!

Reply:
Ahh yes didn't think of something so simple. . .  Thanks!